modules/social_features/social_group/modules/social_group_request/social_group_request.module

```diff
@@ -299,29 +300,31 @@ function social_group_request_preprocess_activity(array &$variables) {
   $activity = $variables['elements']['#activity'];
 
   if (!$activity->get('field_activity_message')->isEmpty()) {
+    $message_storage = \Drupal::entityTypeManager()->getStorage('message');
     /** @var Drupal\message\Entity\Message $message */
     $message = $activity->field_activity_message->entity;
 
-    if (!$message->get('template')->isEmpty()) {
-      $message_template = $message->template->target_id;
-
-      if ($message_template === 'request_to_join_a_group') {
-        $activity_entity = $activity->getRelatedEntity();
-
-        if ($activity_entity instanceof GroupContentInterface) {
-          $variables['full_url'] = Url::fromRoute('view.group_pending_members.membership_requests', [
-            'arg_0' => $activity_entity->getGroup()->id(),
-          ]);
-        }
-      }
-
-      if ($message_template === 'approve_request_join_group') {
-        $activity_entity = $activity->getRelatedEntity();
-
-        if ($activity_entity instanceof GroupContentInterface) {
-          $variables['full_url'] = Url::fromRoute('social_group.stream', [
-            'group' => $activity_entity->getGroup()->id(),
-          ]);
+    if ($message instanceof Message) {
+
+      if (!$message->get('template')->isEmpty()) {
```

Comment:
Please merge these conditions into one:
`if ($message instanceof Message && !$message->get('template')->isEmpty()) {`

P.S. No need to add a blank line after checking